mods/cnc/chrome/connection.yaml

```diff
@@ -152,7 +152,7 @@ Container@CONNECTION_SWITCHMOD_PANEL:
 			Y: 119
 			Width: 140
 			Height: 35
-			Text: Abort
+			Text: Cancel
```

Comment:
ABORT_BUTTON Text: Cancel?

Reply:
it's called cancel in RA yaml

Reply:
I mean it is inconsistent towards the ID.

Reply:
renamed the one in common to abort

Reply:
<img width="510" alt="Screenshot 2023-03-17 at 09 45 02" src="https://user-images.githubusercontent.com/37534529/225843592-25fe762a-7ef7-4c97-b708-1725f675070f.png">

Thought maybe `Quit` would be better